tests/test_integration.py

```diff
@@ -39,59 +39,6 @@ def tearDown(self):
 
         self.client_patcher.stop()
 
-    def test_dns_failure(self):
-        api_base = stripe.api_base
-        try:
-            stripe.api_base = 'https://my-invalid-domain.ireallywontresolve/v1'
-            self.assertRaises(stripe.error.APIConnectionError,
-                              stripe.Customer.create)
-        finally:
-            stripe.api_base = api_base
-
-    def test_run(self):
```

Comment:
@ob-stripe I removed this one because it did not seem to do anything that was not done in test_charge.py. Let me know if that was wrong.

Reply:
No, that seems fine. A lot of the tests in this file were no longer useful.